src/chat/components/leftbar/OptionsModal.js

```diff
@@ -84,33 +84,34 @@ class OptionsModal extends React.Component {
     }
     
     handleDeleteRender = () => {
-        if(!this.state.isDeleteConfirmOpen){
-           return(
-                <Button 
-                    style={styles.deleteButton} 
-                    value="Delete"  
-                    onClick={() => this.setState({isDeleteConfirmOpen: true,})}
-                />
-            )
-        } else {
-            return (
-                <div>
-                    <p style={styles.p}>Are you sure?</p>
-                    <Button 
-                        style={styles.confirmButton} 
-                        value="Yes"  
-                        onClick= {() => this.props.dispatch(deleteChat(this.props.chat))}
-                    />
+        if(this.props.chat.isTwoPeople || this.props.chat.creator === currentUser._id || currentUser.isAdmin){
```

Comment:
`currentUser.isAdmin` is a function that returns a boolean, not a boolean.

Also, this line is a bit long and should be split up.